.docker/ci/Dockerfile

```diff
@@ -6,9 +6,8 @@ MAINTAINER Dave Coleman dave@picknik.ai
 
 ENV TERM xterm
 
-# Setup catkin workspace
-ENV CATKIN_WS=/root/ws_moveit
```

Comment:
I don't like how `/root/ws_moveit` is now spread across multiple files. Wouldn't it be better to set

```dockerfile
ENV CATKIN_WS=/root/ws_moveit
ENV ROS_UNDERLAY=$CATKIN_WS/install
```

Reply:
The `CATKIN_WS` folder created/used in the `ci` and `ci-shadow-fixed` containers is only temporary: It's deleted after installing dependencies via rosdep. This folder could go anywhere.
Only the `source` container actually creates a workspace. Hence, it is sufficient to create it there.

Reply:
@mlautman Can we proceed with this PR? I would like to use the `source` docker container for MTC as well.